src/Projects/Cida.Api/Models/Filesystem/DirectoryItem.cs

```diff
@@ -24,16 +25,36 @@ public DirectoryItem(string name, Directory directory)
             this.Directory?.InternalFiles.Add(this);
         }
 
-        public void Move(Directory directory)
+        public void Move(Directory directory, bool moveParent = false)
```

Comment:
That seems kinda weird to me, moving the parent directory although you call it on a child item...

Reply:
This is so the filestructure won't be changed. The FTP-Client wants to move everything under his directory but you can only upload a file and therefor it will only move the file and the whole structure above the file gets lost. With this the structure is still in place and it's ensured that the module is in the correct root folder